packages/test-synthetix/test.sh

```diff
@@ -0,0 +1,6 @@
+git clone git@github.com:Synthetixio/synthetix.git
```

Comment:
This needs to have `#!/bin/bash` at the top

Reply:
Also you should make it executable by running `chmod +x test.sh` and then commit the file